pkg/scheduler/apis/config/testing/compatibility_test.go

```diff
@@ -1824,8 +1964,13 @@ func TestPluginsConfigurationCompatibility(t *testing.T) {
 					{Name: "ImageLocality", Weight: 24},
 					{Name: "NodeResourcesBalancedAllocation", Weight: 24},
 				},
-				"BindPlugin": {{Name: "DefaultBinder"}},
+				"ReservePlugin":   {{Name: "VolumeBinding"}},
+				"UnreservePlugin": {{Name: "VolumeBinding"}},
+				"PreBindPlugin":   {{Name: "VolumeBinding"}},
+				"BindPlugin":      {{Name: "DefaultBinder"}},
+				"PostBindPlugin":  {{Name: "VolumeBinding"}},
 			},
+			wantPluginConfig: defaultPluginConfigs,
```

Comment:
I didn't mean that this test it's wrong, it's not :)

I meant that there should be another test for when `KubeSchedulerConfiguration.BindTimeout` is used, it's corresponding PluginConfig should be created. Correct?

Reply:
I see. This can't be tested in existing tests of this package (`TestCompatibility_v1_Scheduler` tests policy configs but `KubeSchedulerConfiguration.BindTimeoutSeconds` is not a part of the policy).

I've added a new test `TestInjectingPluginConfigForVolumeBinding` in `scheduler_test.go` which can be removed   when we eliminate `KubeSchedulerConfiguration.BindTimeoutSeconds`.

Reply:
you can still do it, on ```scheduler.New``` you can add the option ```scheduler.WithBindTimeoutSeconds(bindTimeoutSeconds)``` to pass in the test timeout, and the following option to test the config that gets generated:

```
scheduler.WithBuildFrameworkCapturer(func(p config.KubeSchedulerProfile) {
	if diff := cmp.Diff(tc.wantPluginConfig, p.PluginConfig); diff != "" {
		t.Errorf("unexpected plugins diff (-want, +got): %s", diff)
			}
		})
```

Reply:
but may be it is not necessary if you have coverage in scheduler_test.go since those will be removed anyway in the follow up PR.

Reply:
yes, it will be removed anyway, so I add a separate test.

Reply:
sg